### DISCUSSION TITLE
GNOME-software apps not showing up in dmenu

### DISCUSSION THREAD

**Discussion by Martie06** [i3/i3#5688] [Q&A]
I recently installed i3 on my Zorin OS, and installed apps like Chrome, Geogebra etc. through the os-build-in GNOME Software Center that uses snap. The applications appear in the default wm, but in i3 they don't show up in dmenu. I can still launch them through the GNOME Software Center, but that's obviously unpractical and takes the whole meaning of i3 being efficient away. 

What can I do to make the apps appear in dmenu?

**Comment by orestisfl** [comment]
Do they not show up in `i3-dmenu-desktop`?

**Reply by Martie06** [reply]
Thanks for responding!
Didn't know about the command, but the apps do show up there. Is it possible to launch them in / move them to the default dmenu?

**Reply by orestisfl** [reply]
Is it possible that Gnome Software installs apps through flatpak/snap and for that reason they are not available in $PATH?

You can check by running
```
find / -iname 'chrom*.desktop' 2>/dev/null
```
and then paste the content of the file(s) that show up.
